src/toil/wdl/wdl_functions.py

```diff
@@ -36,10 +36,40 @@
 
 class WDLRuntimeError(Exception):
     """ WDL-related run-time error."""
+
     def __init__(self, message):
         super(WDLRuntimeError, self).__init__(message)
 
 
+class WDLPair:
+    """
+    Represent a WDL Pair literal defined at
+    https://github.com/openwdl/wdl/blob/main/versions/development/SPEC.md#pair-literals
+    """
+    # TODO: figure out placement for these classes.
```

Comment:
Can `WDLPair` and `WDLPairType` go together somewhere? maybe in a `wdl_types.py`? If they're far apart it can be confusing that we have one to represent the type and one to represent the instance.

If we were really cool, we would figure out how to set things up so that `WDLPairType(WDLIntType, WDLStringType)` was itself a type, and a WDL pair object would be an instance of it, so we wouldn't need a separate `WDLPair`. We could say `WDLPairType(WDLIntType, WDLStringType)(5, 'cows')` for example. I think this would involve metaclasses.  But that might be over-engineered, so don't do it unless the spirit moves you.